zombienet/tests/0007-full_node_warp_sync.feature

```diff
@@ -0,0 +1,12 @@
+Description: Full node catching up using warp sync
+Network: ./0007-full_node_warp_sync.toml
+Creds: config
+
+alice: is up
+bob: is up
+charlie: is up
+dave: is up
+eve: is up
+alice: parachain 2000 is registered within 225 seconds
+dave: reports block height is at least 7 within 250 seconds
+eve: reports block height is at least 7 within 250 seconds
```

Comment:
I am not entirely sure how warp sync works for collators, but for relay-chain 7 blocks is not enough to perform the warp-sync.

Please refer to [warp-sync description](https://github.com/paritytech/substrate/blob/8a3b2f2e0d4bcf9614dbf174ad2b24da6f9b0b44/client/network/README.md?plain=1#L240-L257).

As warp sync works by syncing the changes in validators set, which is updated at the end of era, we would need more eras in the test.

Building such long chain on the fly in the CI does not make sense, so we introduced possibility of [re-using the DB snapshot](https://github.com/paritytech/substrate/blob/master/zombienet/0001-basic-warp-sync/test-warp-sync.toml#L14) for the nodes.

[Here](https://github.com/paritytech/substrate/blob/master/zombienet/0001-basic-warp-sync/README.md) is a description of how the warp-sync zombienet was prepared for relaychain. Maybe this is a start point for building similar solution for parachains.

Reply:
Thanks for the update @skunert @michalkucharczyk 
I amended the test based on the suggestions, let me know if it is acceptable